src/status_im/contexts/communities/overview/style.cljs

```diff
@@ -18,6 +19,9 @@
    :background-color background-color
    :height           (+ 20 ;; Area hidden by sheet on top but visible with rounded borders
                         92
+                        ;; On Android we count the navigation bar page-nav padding top
+                        ;; because it isn't overlapped with the safe-area top.
+                        (when platform/android? 12)
                         safe-area/top)})
```

Comment:
The safe area fix on Android :eyes: the community-cover image now is bigger and looks better :)